docs/ROADMAP.md

```diff
@@ -13,8 +13,8 @@ There is an urgent, global need for an official WHO App to help contain and miti
 Version numbering terminology:
 
 * **v0.1**: refers to the initial buildout that will be submitted to the app stores but not intended for public release, aiming for submission on Thursday, March 26, 2020 ([v0.1 milestone](https://github.com/WorldHealthOrganization/app/milestone/2))
-* **v0.2**: an informal preview build for WHO by end of day Friday, March 27, 2020.  This is not a separate milestone.
-* **v1.0**: first public release, sent to WHO end of day Monday, March 30, 2020 ([v1.0 milestone](https://github.com/WorldHealthOrganization/app/milestone/4))
+* **v1.0**: public release on the app stores, aiming for submission to app stores on Monday, March 30, 2020 ([v1.0 milestone](https://github.com/WorldHealthOrganization/app/milestone/4))
+* **vNext**: describes features considered for future releases, but not yet prioritized into a specific milestone ([vNext milestone](https://github.com/WorldHealthOrganization/app/milestone/3))
 
 ### v0.1
```

Comment:
v0.1 should be moved to the Release Notes and marked "not publicly released"

You can also use GitHub's release process with notes, for example: https://github.com/google/j2objc/releases - but that's better suited to a library that's pulled in as a dependency. For this, I think a single file with release notes seems more appropriate. Not sure where the file should be and how it should be named. Maybe just move it to the bottom of the ROADMAP?

Reply:
I've put this at the bottom of ROADMAP for now, we can move it out to a separate Release Notes file in the future or use the tags page